src/parser/model/cloudformation-resource.ts

```diff
@@ -29,6 +29,13 @@ export class CloudFormationResource extends Resource {
         this.bindings = this.resource.OrganizationBindings as IOrganizationBindings;
         if (this.bindings) {
             this.includeMasterAccount = this.bindings.IncludeMasterAccount;
+
+            if (Array.isArray(this.bindings.ExcludeAccounts)) {
+                this.excludeAccounts = this.bindings.ExcludeAccounts as Array<Reference<AccountResource>>;
+            } else {
+                this.excludeAccounts = [ this.bindings.ExcludeAccounts as Reference<AccountResource> ];
```

Comment:
@tstroband i dont believe this is right. the typeof this.binding.ExcludeAccount is IResourceRef (string | {Ref: string}) not Reference.